python/ray/experimental/named_actors.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
```

Comment:
Let's reserve assertions for checking things that should never actually happen. The errors (in both `register_actor` and `get_actor`) can happen if the user passes in the wrong values, so we should raise some sort of exception

Reply:
Handled.